src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
```

Comment:
Well that's the thing--I don't think that Enum or even Mabe/Enum has any utility without being extended. In fact, the base class itself is indeed abstract: 

https://github.com/marc-mabe/php-enum/blob/master/src/Enum.php#L16

So I think you're right on the money on this one